modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/ShelfFragment.kt

```diff
@@ -276,63 +286,50 @@ class ShelfAdapter(val editable: Boolean, val listener: ((ShelfItem, Boolean) ->
                 start()
             }
         }
-    }
 
-    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
-        val inflater = LayoutInflater.from(parent.context)
-        return when (viewType) {
-            R.layout.adapter_shelf_item -> {
-                val binding = AdapterShelfItemBinding.inflate(inflater, parent, false)
-                ItemViewHolder(binding)
-            }
-            R.layout.adapter_shelf_title -> {
-                val binding = AdapterShelfTitleBinding.inflate(inflater, parent, false)
-                TitleViewHolder(binding)
+        fun bind(item: ShelfItem) {
+            composeView.setContent {
+                AppTheme(theme.activeTheme) {
```

Comment:
```suggestion
                AppTheme(Theme.ThemeType.DARK) {
```

As the player is always a dark theme otherwise the select highlight colours are too dark. 

| Before | After |
| --- | --- |
| ![Screenshot_20241113_114903](https://github.com/user-attachments/assets/4ddc0845-fccc-4981-88ca-882708c6aee9) | ![Screenshot_20241113_114723](https://github.com/user-attachments/assets/fadc4768-f7b0-46bf-8d9e-1787c7ee9438) |

Reply:
Changing the theme to dark may cause other side effects: 
Before | After
---|----
![image](https://github.com/user-attachments/assets/0e44d764-7c0d-4ca0-b8eb-db7da5970d74) | ![image](https://github.com/user-attachments/assets/0bf0ac10-72bd-412c-b123-9038bbdefbfa)

Let me test it a bit more.